src/ep_worker.c

```diff
@@ -9823,6 +9824,11 @@ static ep_stat_t w_form_query_getall_insert_new(worker_data_t *wd, obj_info_t *o
     tmp_cnt = 0;
     for (i = 0; i < parsed_backend_str->subst_val_num; i++)
     {
+        if (parsed_backend_str->subst_val[i].mmx_subst_val.leaf_name, idx_params[idx_params_num - 1]) {
```

Comment:
I don't understand the goal of expression like
```C
if (a, b) {
   // do something
}
```
Why not use
```C
if (idx_params[idx_params_num - 1]) {
```

Reply:
I believe it was `if (0 == strcmp(....))`

Reply:
fixed